packages/cli/src/commands/env/index.ts

```diff
@@ -129,7 +120,7 @@ export default async function main(ctx: NowContext) {
     return 1;
   } else {
     const { project, org } = link;
-    client.currentTeam = org.type === 'team' ? org.id : undefined;
+    config.currentTeam = org.type === 'team' ? org.id : undefined;
```

Comment:
Whats the difference between `client.currentTeam` and `client.config.currentTeam`?

Reply:
`client.currentTeam` no longer exists, instead it takes the entire "global config" object as `client.config`.